src/init/starship.ps1

```diff
@@ -106,7 +135,16 @@ $null = New-Module starship {
         $arguments += "--status=$($lastExitCodeForPrompt)"
 
         # Invoke Starship
-        $promptText = Invoke-Native -Executable ::STARSHIP:: -Arguments $arguments
+        $promptText = if ($script:TransientPrompt) {
+            $script:TransientPrompt = $false
+            if (Test-Path function:Invoke-Starship-TransientFunction) {
+                Invoke-Starship-TransientFunction
+            } else {
+                "> "
```

Comment:
I think I'd prefer going with the default style and symbol of the character module here, but on the other hand, that could mislead people into thinking this is the character module.

Reply:
I would have gone with the character module too, but unless https://github.com/starship/starship/pull/3442#issuecomment-1024270805 is resolved, I'd really prefer avoiding another starship invocation here.

Reply:
I meant something like this: ``"`e[1;32m❯`e[0m"`` instead of `starship module character`

Reply:
Oh, yeah that I can add